support/cas-server-support-rest/src/main/java/org/apereo/cas/support/rest/resources/TicketGrantingTicketResource.java

```diff
@@ -60,7 +60,7 @@
      * @return ResponseEntity representing RESTful response
      */
     @PostMapping(value = "/v1/tickets", consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE)
-    public ResponseEntity<String> createTicketGrantingTicket(@RequestBody final MultiValueMap<String, String> requestBody,
+    public ResponseEntity<String> createTicketGrantingTicket(@RequestBody(required=false) final MultiValueMap<String, String> requestBody,
```

Comment:
Why the change?

Reply:
In the mechanism I used in 3.6, our X509 authentication requests didn't contain a body, it was just a POST request with the client authentication.  Without this required=false, spring throws an exception as the RequestBody is required by default.